app/src/main/java/com/bogdan/codeforceswatcher/util/Analytics.kt

```diff
@@ -16,4 +16,22 @@ object Analytics {
         }
     }
 
+    fun logUsersListRefresh() {
+        if (isEnabled) {
+            FirebaseAnalytics.getInstance(CwApp.app).logEvent("users_list_refresh", Bundle())
```

Comment:
I think, it's better to extract `FirebaseAnalytics.getInstance(CwApp.app)` to class field and then reuse it in all methods.